pallets/runtime/tests/src/compliance_manager_test.rs

```diff
@@ -241,6 +241,17 @@ fn should_add_and_verify_asset_rule_we() {
     assert_eq!(result.rules[0].sender_rules[0].rule, sender_rule);
     assert_eq!(result.rules[0].receiver_rules[0].rule, receiver_rule1);
     assert_eq!(result.rules[0].receiver_rules[1].rule, receiver_rule2);
+
+    for _ in 0..2 {
+        ComplianceManager::add_active_rule(
+            token_owner_signed.clone(),
+            ticker,
+            vec![sender_rule.clone()],
+            vec![receiver_rule1.clone(), receiver_rule2.clone()],
+        )
+    }
+    assert_ok!(ComplianceManager::remove_active_rule(1)); // OK; latest == 3
+    assert_err!(ComplianceManager::remove_active_rule(1)); // BAD OK; latest == 3, but 1 was just removed.
```

Comment:
```suggestion
    // ERROR: latest == 3, but 1 was just removed.
    assert_noop!(ComplianceManager::remove_active_rule(1), Error::InvalidRuleId);
```
(Let's also test that we don't have storage modification and the specifics of the error.)

Reply:
Alright will make the change